synd-mchain/src/methods/eth_methods.rs

```diff
@@ -31,6 +31,9 @@ use std::{
     time::UNIX_EPOCH,
 };
 
+/// hardfork timestamp
+pub const HARDFORK_TS: u64 = 1764565200;
```

Comment:
we should probably name this a bit more explicitly like: `L1_BLOCK_NUM_HARDFORK_TS`